site/toc.js

```diff
@@ -116,6 +117,9 @@ async function createToc(subDir) {
 
 		if (entry.isDirectory) {
 			entry.children = await createToc(srcPath);
+			entry.children.sort(
```

Comment:
I've been thinking about this for a time, I wonder if we need to come up with some other method for sorting ...

This will work for now though 👍